emails/tests/views_tests.py

```diff
@@ -969,6 +969,15 @@ def test_reply_with_emoji_in_text(self) -> None:
             expected_fixture_name="s3_stored_replies_with_emoji",
         )
 
+    def test_reply_case_insensitive_relay_user_email(self) -> None:
+        """Casing of a Relay user's email doesn't prevent outgoing replies"""
+        self.user.email = "Source@sender.com"
```

Comment:
quibble (non-blocking): can we try a few more variants here just to make sure it's not some weird case where a first capital letter works but a capital letter somewhere else doesn't?

Reply:
I'm just going to update this to use a different (non-capitalized) casing approach, so that I don't have to extend `successful_reply_test_implementation` or add mostly redundant cases to the suite. It should make sure that we don't accidentally remove `.lower()` in favor of a potential `.capitalize()` logic.